docs/web3.eth.rst

```diff
@@ -403,7 +403,8 @@ The following methods are available on the ``web3.eth`` namespace.
             'from': '0xa1e4380a3b1f749673e270229993ee55f35663b4',
             'gasUsed': 21000,
             'logs': [],
-            'root': '96a8e009d2b88b1483e6941e6812e32263b05683fac202abc622a3e31aed1957',
+            'logsBloom': '0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,
```

Comment:
This error seems like it could be caught were these code blocks to be doctested.  Is that something you'd be up for adding?  It should be reasonably simple, though it will require setting up a bit of chain state in a `.. testsetup` block.

Reply:
@pipermerriam when you get a chance, can you take a look at 72d4098. I believe that does what we want. I skipped the "not yet mined" line in the doctest on 383ee34. I didn't use a `.. testsetup` block, so I am not sure I accomplished exactly what you want.

It is passing locally but CircleCI is failing [here](https://circleci.com/gh/ethereum/web3.py/41377?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) with `web3.exceptions.UnhandledRequest: No providers responded to the RPC reque`. This is weird, because the web3 setup seems identical to the rest of the docs. Any idea there?